### PR TITLE
fix/remove broken links

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: docs/architecture/microservices/multi-container-microservice-net-applications/implement-api-gateways-with-ocelot.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ The important point here for Ocelot is the `configuration.json` file that you mu
 
 There are two sections to the configuration. An array of Re-Routes and a GlobalConfiguration. The Re-Routes are the objects that tell Ocelot how to treat an upstream request. The Global configuration allows overrides of Re-Route specific settings. It’s useful if you don’t want to manage lots of Re-Route specific settings.
 
-Here’s a simplified example of [ReRoute configuration file](https://github.com/dotnet-architecture/eShopOnContainers/blob/dev/src/ApiGateways/Web.Bff.Shopping/apigw/configuration.json) from one of the API Gateways from eShopOnContainers.
+Here’s a simplified example of [ReRoute configuration file](https://github.com/dotnet-architecture/eShopOnContainers/blob/master/src/ApiGateways/Web.Bff.Shopping/apigw/configuration.json) from one of the API Gateways from eShopOnContainers.
```
Comment (Youssef1313) [inline docs/architecture/microservices/multi-container-microservice-net-applications/implement-api-gateways-with-ocelot.md]: The `dev` branch is the default one for that repository. If that file was deleted from that branch, I think there should be a reason for that. Hence, the article may be needed to get updated in a different way.

Comment (BillWagner) [reply docs/architecture/microservices/multi-container-microservice-net-applications/implement-api-gateways-with-ocelot.md]: We should check with @nishanil or @CESARDELATORRE on that question.

**Review by BillWagner — APPROVED** [review]
This LGTM @mairaw 

You can :shipit: once we have the answer on the samples link.

**Review by Thraka — APPROVED** [review]
Approved my owned articles. Thanks!

**Review by Thraka — CHANGES_REQUESTED** [review]
The download link for the .NET Core uninstall tool should be replaced with a different one, as it's the download link on the page. @mlacouture is this tool still usable? Where do customers download it?

**Comment by mairaw** [comment]
@Thraka it's @sfoslund who owns that. I could change that to https://github.com/dotnet/cli-lab/releases, but I thought that folks should actually go to the article to see where to get the tools from and how to use them. The index page is just a list of articles in that section.

**Comment by Thraka** [comment]
@mairaw  I think that releases page looks good. I just don't want users who find this article by search to then not know where to go to download the tool.

File: docs/architecture/microservices/multi-container-microservice-net-applications/implement-api-gateways-with-ocelot.md (hunk 1 above)
Comment (nishanil) [reply docs/architecture/microservices/multi-container-microservice-net-applications/implement-api-gateways-with-ocelot.md]: We've dropped the support for Ocelot in eShop code but will retain the documentation in the book. As part of 3.1 updates (https://github.com/dotnet/docs/pull/16202), this guide will be updated to reflect changes.  For now, the redirection to master is correct.
@mvelosop  Can you note this and update the link in your pr?

**Review by Thraka — APPROVED** [review]

**Pull request merged**